lib/scenario_runner.py

```diff
@@ -393,7 +392,6 @@ def recursive_lookup(k, d):
         if '/' in self._original_filename:
             self.__working_folder_rel = self._original_filename.rsplit('/', 1)[0]
             self.__working_folder = usage_scenario_file.rsplit('/', 1)[0]
-            #self.__filename = usage_scenario_file.rsplit('/', 1)[1] # we currently do not use this variable
```

Comment:
why remove that? I find comments like this which might indicate possible future implementations quite useful ...